.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
-
+.idea/
```

Comment:
It is better to move it to your local git ignore on system level

https://gist.github.com/subfuzion/db7f57fff2fb6998a16c